bm-instance-agent/bm_instance_agent/api/middleware.py

```
@@ -45,7 +45,7 @@ def replacement_start_response(status, headers, exc_info=None):
         if (state['status_code'] // 100) not in (2, 3):
             app_iter = [i.decode('utf-8') for i in app_iter]
             body = [jsonutils.dumps({'error_message': '\n'.join(app_iter)})]
-            body = [i.encode('utf-8') for i in body]
+            body = [i.encode('utf-8') if type(i) == bytes else i for i in body]
```

Ruff (0.12.2)

Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)